src/rgw/rgw_rados.h

```diff
@@ -2548,6 +2548,10 @@ class RGWMPObj {
     meta = "";
     upload_id = "";
   }
+  friend std::ostream& operator<<(std::ostream& out, const RGWMPObj& obj) {
+    return out << "RGWMPObj:{ prefix=\"" << obj.prefix <<
```

Comment:
nit, could put
```c++
return out << "RGWMPObj:{ prefix=" << std::quoted(obj.prefix)
     << ", meta=" << std::quoted(obj.meta) << " }";
```

Reply:
I did not know about std::quoted -- thanks!

Reply:
After trying out std::quoted in a few places, it appears it only works if the parameter is essentially a string; if it is ostream'able it produces an error. Oh well.

Reply:
yeah, that's a pain. probably we could have our own `ceph::quoted()` . fortunately, `std::quoted` works in this case.